### PR TITLE
Bump deps on -specs and -base

### PR DESCRIPTION
**Pull request by edsko**

### CODE REVIEW

File: ouroboros-consensus-shelley-test/src/Test/Consensus/Shelley/Generators.hs [hunk 1]
```diff
@@ -420,9 +420,13 @@ instance Crypto c => Arbitrary (SL.StakeReference c) where
   arbitrary = genericArbitraryU
   shrink    = genericShrink
 
+
```
Comment (mrBliss) [inline ouroboros-consensus-shelley-test/src/Test/Consensus/Shelley/Generators.hs:423]:
```suggestion
```

File: cabal.project [hunk 2]
```diff
@@ -132,98 +132,98 @@ source-repository-package
 source-repository-package
   type: git
   location: https://github.com/input-output-hk/cardano-base
-  tag: 9b82346b78e212fcd3e795faf9dd7f3266ab7297
+  tag: 8061c5dd00f7aad134b18a4e6818879641f8589f
   --sha256: 04iv32x233bck4xd34x2g0yaxj438f57d3z6m0pvvzj2yca1slxm
```
Comment (mrBliss) [inline cabal.project:136]: I suppose you'll use the output from CI to update the sha256 sums?

Comment (edsko) [reply cabal.project:136]: That's the plan :)

**Comment by edsko** [comment]
bors merge

**Comment by edsko** [comment]
bors merge

**Comment by edsko** [comment]
bors merge

**Comment by edsko** [comment]
bors merge

**Pull request merged**